cli/cobra.go

```diff
@@ -57,6 +59,14 @@ func operationSubCommands(cmd *cobra.Command) []*cobra.Command {
 	return cmds
 }
 
+func wrappedFlagUsages(cmd *cobra.Command) string {
+	width := 80
```

Comment:
I'm not an expert on terminal/curses stuff, but isn't there a way to query for the terminal width?